xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -739,6 +739,9 @@ platform.core.profile.watchlist.unwatch=Remove from my watch list
 core.footer.creation=Created by {0} on {1}
 core.footer.translationCreation=Translated into {0} by {1} on {2}
 core.footer.modification=Last modified by {0} on {1}
+
+core.document.header.info.label=Page header
+core.document.header.menu.nav.label=Page actions
```

Comment:
I think it would be good to have an explanation for these strings as comments, in particular as these are not normally shown to the user.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2133/commits/065f12c47602b8e7eca5dd69fec16652290aa38b